Broken with pytest 6.0.1

Throws an exception
```
node = <Function test_auth>

    def get_default_snapshot_dir(node):
        """
        Returns the default snapshot directory for the pytest test.
    
        :type node: _pytest.python.Function
        :rtype: Path
        """
        test_module_dir = node.fspath.dirpath()
        test_module = node.fspath.purebasename
        if node.originalname is None:
            test_name = node.name
            parametrize_name = None
        else:
            test_name = node.originalname
            parametrize_match = PARAMETRIZED_TEST_REGEX.match(node.name)
>           assert parametrize_match is not None, 'Expected request.node.name to be of format TEST_FUNCTION[PARAMS]'
E           AssertionError: Expected request.node.name to be of format TEST_FUNCTION[PARAMS]
E           assert None is not None

```

--- a/README.rst
+++ b/README.rst
@@ -105,7 +105,7 @@ In the second example, the developer would simply
 Snapshot testing can be used for expressions whose values are strings.
 For other types, you should first create a *human readable* textual representation of the value.
 For example, to snapshot test a *json-serializable* value, you could either convert it into json
-or preferably convert it into the more readable yaml format using `PyYaml`_:
+or preferably convert it into the more readable yaml format using `PyYAML`_:
 
 .. code-block:: python
 

--- a/pytest_snapshot/plugin.py
+++ b/pytest_snapshot/plugin.py
@@ -230,7 +230,7 @@ def get_default_snapshot_dir(node):
     """
     test_module_dir = node.fspath.dirpath()
     test_module = node.fspath.purebasename
-    if node.originalname is None:
+    if '[' not in node.name:
         test_name = node.name
         parametrize_name = None
     else:

--- a/tox.ini
+++ b/tox.ini
@@ -2,11 +2,11 @@
 [tox]
 envlist =
     py27-pytest{3,4}-coverage,
-    py35-pytest{3,4,5,}-coverage,
-    py36-pytest{3,4,5,}-coverage,
-    py37-pytest{3,4,5,}-coverage,
-    py38-pytest{3,4,5,}-coverage,
-    pypy3-pytest{3,4,5,}-coverage,
+    py35-pytest{3,4,5,6,}-coverage,
+    py36-pytest{3,4,5,6,}-coverage,
+    py37-pytest{3,4,5,6,}-coverage,
+    py38-pytest{3,4,5,6,}-coverage,
+    pypy3-pytest{3,4,5,6,}-coverage,
     flake8
 
 [testenv]
@@ -14,6 +14,7 @@ deps =
     pytest3: pytest >=3, <4
     pytest4: pytest >=4, <5
     pytest5: pytest >=5, <6
+    pytest6: pytest >=6, <7
     pytest: pytest
 
     coverage: coverage

